README.VisualStudio.md

```diff
@@ -2,16 +2,44 @@
 
 Here are some field notes for building `playd` with Microsoft Visual Studio.
 
-Visual Studio builds used to be much harder to pull off, but nowadays most
-of the dependencies are nicely packaged.  _Most_.
+Visual Studio builds used to be much harder to pull off, but nowadays all
+of the dependencies are nicely packaged for both x86 and x64.
+
+**Note**: The x64 build currently has a few warnings, but it appears to work!
 
 ## Visual Studio Versions
 
 At time of writing, `playd` has been tested with Visual Studio 2015 Update 3.
-As `playd` needs a C++11 compiler, earlier versions will likely fail;
+As `playd` needs a C++14 compiler, earlier versions will likely fail;
 newer versions may work, but this is not guaranteed.
 
-## Assembling Libraries and Includes
+## The Easy Way: WindowsBuilder.ps1
+
+Now you can sit back and relax whilst this script downloads the binary versions of all the libraries.
+
+Usage: `.\WindowsBuilder.ps1 [-deps] [-playd] -arch x86|x64 [<CommonParameters>]`
+
+Full help text: `Get-Help .\Windows-Builder.ps1`
+
+The build will be in a directory like: `x86\build\Release\` along with all the necessary DLLs.
+
+**Note**: Only the **Release** configuration is currently supported.
+Until that's supported, you're on your own. Use Linux or something.
```

Comment:
"Until that's supported": you haven't actually mentioned what 'that' refers to - a better line would be something like "If you want to use other configurations, you'll have to use Linux or something."